parsing/tests/parsing.spec.ts

```diff
@@ -236,5 +254,54 @@ describe('common', () => {
             expect(recipe.building.maxPower).toBe(2000);
             expect(recipe.isAlternate).toBe(false);
         });
+
+
+        it('validate a biomass power generation recipe (solid biomass)', () => {
+            //arrange
+            const recipe : Recipe = results.powerGenerationRecipes.find((item: { id: string; }) => item.id === 'GeneratorBiomass_Automated_Biofuel');
+
+            //act
+
+            //assert
+            expect(recipe).toBeDefined();
+            expect(recipe.displayName).toBe('Biomass Burner (Solid Biofuel)');
+            expect(recipe.ingredients.length).toBe(1);
+            expect(recipe.ingredients[0].part).toBe('Biofuel');
+            expect(recipe.ingredients[0].amount).toBe(0.06666666666666667);
+            expect(recipe.ingredients[0].perMin).toBe(4);
+            expect(recipe.products.length).toBe(0);
+            expect(recipe.building.name).toBe('BiomassBurner');
+            expect(recipe.building.power).toBe(30);
+            expect(recipe.isAlternate).toBe(false);
+            expect(recipe.isPowerGenerator).toBe(true);
+        });
+
+        it('validate a nuclear power generation recipe (Uranium Fuel Rod)', () => {
+            //arrange
+            const recipe : Recipe = results.powerGenerationRecipes.find((item: { id: string; }) => item.id === 'GeneratorNuclear_NuclearFuelRod');
+
+            //act
+
+            //assert
+            expect(recipe).toBeDefined();
+            expect(recipe.displayName).toBe('Nuclear Power Plant (Uranium Fuel Rod)');
+            expect(recipe.ingredients.length).toBe(2);
+            expect(recipe.ingredients[0].part).toBe('NuclearFuelRod');
+            expect(recipe.ingredients[0].amount).toBe(0.2/60);
+            expect(recipe.ingredients[0].perMin).toBe(0.2);
+            expect(recipe.ingredients[1].part).toBe('Water');
+            expect(recipe.ingredients[1].amount).toBe(4);
+            expect(recipe.ingredients[1].perMin).toBe(240);
+            expect(recipe.products.length).toBe(1);
+            expect(recipe.products[0].part).toBe('NuclearWaste');
+            expect(recipe.products[0].amount).toBe(0.8333333333333334);
+            expect(recipe.products[0].perMin).toBe(50);
+            expect(recipe.products[0].isByProduct).toBe(true);
+            expect(recipe.building.name).toBe('NuclearPowerPlant');
+            expect(recipe.building.power).toBe(2500);
+            expect(recipe.isAlternate).toBe(false);
+            expect(recipe.isPowerGenerator).toBe(true);
+        });
+
```

Comment:
Should probably add another test for Biomas burners for Wood, just in case we somehow break a number of products burners can serve.

Reply:
done!